tests/benchmark/test_custream.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+from hpcbench.benchmark.custream import CUDAStream
+from . benchmark import AbstractBenchmarkTest
+
+
+class TestCUDAStream(AbstractBenchmarkTest, unittest.TestCase):
+    EXPECTED_METRICS = dict(
+        copy_bandwidth=746,
```

Comment:
shouldn't the `copy_bandwidth` metric be a float?

Reply:
you asked for this :)
https://github.com/tristan0x/hpcbench/pull/102#discussion_r166939680

Reply:
`copy_bandwidth` is a `MegaBytesPerSecond` metric, declared as a float in `hpcbench/api.py`, so should be the extracted metric.